tests/pytests/common.py

```diff
@@ -134,6 +134,10 @@ def skipOnExistingEnv(env):
     if 'existing' in env.env:
         env.skip()
 
+def skipOnOss(env):
```

Comment:
Did you mean `Standalone` here? Like, not an OSS cluster and not a RLEC cluster?

Reply:
Changed to `SkipOnNonCluster`